scheduler/scheduler-node/src/main/java/org/ow2/proactive/scheduler/task/context/TaskContextVariableExtractor.java

```diff
@@ -122,11 +122,17 @@
 
     public Map<String, Serializable> extractScopeVariables(TaskContext taskContext) {
         Map<String, Serializable> variables = new HashMap<>();
-
+        Map<String, Serializable> previousVariables = null;
+        try {
+            previousVariables = extractVariables(taskContext, false);
```

Comment:
the previousVariables are those the variables from previousTasks? Maybe it would be nice to rename it to previousTasksVariables.

Reply:
they are actually system, job and propagated variables (I am particularly interested in the latter two).